fhir-config/src/main/java/com/ibm/fhir/config/FHIRRequestContext.java

```diff
@@ -94,6 +95,7 @@ public void setTenantId(String tenantId) throws FHIRException {
      * @param base64
      * @throws FHIRException if the given value is not a valid Base64 string
```

Comment:
```suggestion
     * @throws FHIRException if the given value is not a valid Base64 string
     * @deprecated tenantKey should be looked up on-the-fly from the fhir server configuration (e.g. from 
     *                        FHIRDbDAOImpl.getConnection() in fhir-persistence-jdbc)
```

Reply:
I'm going to leave this here to pick up shortly